app/controllers/comments_controller.rb

```diff
@@ -0,0 +1,52 @@
+class CommentsController < ApplicationController
+  before_action :authenticate_user!, only: [:new, :create]
+
+  def index
+    @comments = @review.comments
+  end
+
+  def new
+    @comment = Comment.new
+    @review = Review.find(params[:review_id])
+  end
+
+  def create
+    @comment = current_user.comments.new(comment_params)
+    @review = Review.find(params[:review_id])
+    @sitcom = @review.sitcom
```

Comment:
Is this `@sitcom` variable being used in a view? if not it doesn't need to be an instance variable. Same on line 23.